service/src/service/methods.rs

```diff
@@ -110,7 +109,7 @@ pub async fn key_derive_from_seed(
 ) -> Result<Success, ServiceError> {
     let params = c.params.parse::<KeyDeriveFromSeedParamsAPI>()?;
 
-    let seed: std::vec::Vec<u8> = from_hex_string(params.seed.as_ref())?;
+    let seed = from_hex_string(params.seed.as_ref())?;
```

Comment:
As stated in the function signature, `Result<Vec<u8>, HexDecodeError>`, the unwraped type is already known by the compiler.